core/dbt/cli/main.py

```diff
@@ -453,6 +469,30 @@ def test(ctx, **kwargs):
     return None, True
 
 
+def _get_config_from_ctx(ctx):
+    return RuntimeConfig.from_parts(
+        ctx.obj["project"],
+        ctx.obj["profile"],
+        ctx.obj["flags"],
+    )
+
+
+def _set_manifest_on_task(task, ctx, write_perf_info=False):
+    config = _get_config_from_ctx(ctx)
+    manifest = ManifestLoader.get_full_manifest(config, write_perf_info=write_perf_info)
```

Comment:
`register_adapter(runtimeConfig)` need to be called before/within get_full_manifest